go.mod

```diff
@@ -5,4 +5,5 @@ go 1.13
 require (
 	github.com/mattermost/mattermost-server/v5 v5.39.3
 	github.com/stianeikeland/go-rpio/v4 v4.6.0
+	github.com/thoj/go-ircevent v0.0.0-20210723090443-73e444401d64 // indirect
```

Comment:
Yeah, I just ran 'go get github.com/thoj/go-ircevent' and it added it with the '// indirect'. I searched quickly but didn't find a clear explanation why it might do it in this specific situation, and I stopped looking. (Pure speculation: maybe because status.go is in 'package ledsign' it counts as a separate package and its dependency seems 'indirect'? Not sure.) Anyway, I'll just leave it like this for now but if some Golang guru wants to clean this up properly in the future, that sounds good.